wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -287,6 +375,18 @@ Artikel sollten einfache deutsche Grammatik verwenden.
 
 [Artikel mit informellen Registern](#sprachregister) sollten Semikolons (`;`) nicht verwenden.
 
+### Geviertstrich
+
+Geviertstriche (`—`) müssen mit einem Leerzeichen davor und dahinter verwendet werden. Normale Bindestriche können ebenfalls verwendet werden. Diese Regel trifft nicht auf [Blockzitate](#blockzitate) zu.
```

Comment:
it's better to be consistent than to encourage using both styles

Reply:
We should probably just use the em dashes throughout the wiki, even if they aren't used that much in German otherwise.